lib/iris/analysis/__init__.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2010 - 2016, Met Office
+# (C) British Crown Copyright 2010 - 2017, Met Office
```

Comment:
Are you sure you need to make this change
The build that fails only has two files that have the wrong headers
```
======================================================================

ERROR: test_license_headers (iris.tests.test_coding_standards.TestLicenseHeaders)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/SciTools/iris/lib/iris/tests/test_coding_standards.py", line 345, in test_license_headers

    raise ValueError('There were license header failures. See stdout.')

ValueError: There were license header failures. See stdout.

-------------------- >> begin captured stdout << ---------------------

The file header at lib/iris/analysis/_interpolate_backdoor.py is out of date. The last commit was in 2017, but the copyright states it was 2016.

The file header at lib/iris/analysis/interpolate.py is out of date. The last commit was in 2017, but the copyright states it was 2016.
```

And the PR that introduced this change only modified two files
https://github.com/SciTools/iris/pull/2272/files

Am I missing something?